pypsa/linopf.py

```diff
@@ -906,8 +899,7 @@ def network_lopf(n, snapshots=None, solver_name="cbc",
     #disable logging because multiple slack bus calculations, keep output clean
     snapshots = _as_snapshots(n, snapshots)
     n.calculate_dependent_values()
-    if not skip_pre:
-        n.determine_network_topology()
+    n.determine_network_topology()
```

Comment:
Any particular reason to remove this? I seem to recall a case where you wouldn't want to recalculate the sub_networks.

Reply:
You're right, haven't thought about this. Let's keep it, even though I cannot imagine a case where you would skip it.